CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# How to contribute
+
+Prerequisites:
+
+- Familiarity with [pull requests](https://help.github.com/articles/using-pull-requests) and [issues](https://guides.github.com/features/issues/).
+- Knowledge of [React](https://react.dev/) to create user interfaces.
+
+In particular, this project seeks the following types of contributions:
+
+- **Ideas**: participate in an issue thread or start your own to have your voice
+  heard.
+- **Resources**: submit a pull request to add to RESOURCES.md with links to related content.
+
+# Conduct
+
+We are committed to providing a friendly, safe and welcoming environment for
+all, regardless of personal characteristic.
+
+Please be kind and courteous. There's no need to be mean or rude.
+Respect that people have differences of opinion and that every design or
+implementation choice carries a trade-off and numerous costs. There is seldom
+a right answer, merely an optimal answer given a set of values and
+circumstances.
+
+Please keep unstructured critique to a minimum. If you have solid ideas you
+want to experiment with, make a fork and see how it works.
+
+We will exclude you from interaction if you insult, demean or harass anyone.
+That is not welcome behaviour. We interpret the term "harassment" as
+including the definition in the
+[Citizen Code of Conduct](http://citizencodeofconduct.org/).
+If you have any lack of clarity about what might be included in that concept,
+please read their definition. In particular, we don't tolerate behavior that
+excludes people in socially marginalized groups.
+
+Private harassment is also unacceptable.
+
+Likewise any spamming, trolling, flaming, baiting or other attention-stealing
+behaviour is not welcome.
+
+# Communication
+
+GitHub issues are the primary way for communicating about specific proposed
+changes to this project.
+
+# Frequently Asked Questions
```

Comment:
Do you agree that this is how we should communicate since we cant add everyone to linear? Plus Github will be a great all in one platform for dev and comms

Reply:
I agree this is how it should be done once this unit is completed :)